### PR TITLE
refactor: Populate with Compose Builder from developer-scripts

### PR DESCRIPTION
**Pull request by lenny-intel**
**Just getting Compose Builder and generated compose file moved over for now.**

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
Repo is empty

## What is the new behavior?
Populated with compose builder and ran build to generate all the `pre-release` compose files.
Following where changed:

- Rename the `nexus/nightly-build` release to `pre-release`
- Drop the `compose-files` sub-folder
- Fix get-token in `releases/pre-release/Makefile`  to call new script in compose-builder folder

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?
no

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/.env [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# /*******************************************************************************
+ #  * Copyright 2020 Intel
```
Comment (jim-wang-intel) [inline compose-builder/.env:2]: change copyright to 2021

File: compose-builder/.env [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+VAULT_VERSION=1.5.3
+CONSUL_VERSION=1.9.1
+# Note that Postgres images don't use patch versions.
+POSTGRES_VERSION=12.3-alpine
+REDIS_VERSION=6.0.9-alpine
+KONG_VERSION=2.3-alpine
+KUIPER_VERSION=1.1.1-alpine
+MOSQUITTO_VERSION=1.6.3
+
+RELEASE_FOLDER=../releases/${RELEASE}
```
Comment (jim-wang-intel) [inline compose-builder/.env:51]: move this up together with `RELEASE`, making it easier to read

File: compose-builder/README.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
+
+
```
Comment (jim-wang-intel) [inline compose-builder/README.md:2]: Remove extra lines

File: compose-builder/add-device-bacnet.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-bacnet.yml:2]: change to 2021

File: compose-builder/add-asc-http-export-secure.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-asc-http-export-secure.yml:1]: should we add copyright header?

File: compose-builder/add-asc-http-export.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-asc-http-export.yml:1]: should we add copyright header?

File: compose-builder/add-asc-mqtt-export-secure.yml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-asc-mqtt-export-secure.yml:1]: should we add copyright header?

File: compose-builder/add-asc-mqtt-export.yml [hunk 8]
```diff
@@ -0,0 +1,28 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-asc-mqtt-export.yml:1]: should we add copyright header?

File: compose-builder/add-device-camera.yml [hunk 9]
```diff
@@ -0,0 +1,41 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-camera.yml:2]: 2021

File: compose-builder/add-device-grove.yml [hunk 10]
```diff
@@ -0,0 +1,42 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-grove.yml:2]: 2021

File: compose-builder/add-device-modbus.yml [hunk 11]
```diff
@@ -0,0 +1,40 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-modbus.yml:2]: 2021

File: compose-builder/add-device-mqtt.yml [hunk 12]
```diff
@@ -0,0 +1,43 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-mqtt.yml:2]: 2021

File: compose-builder/add-device-random.yml [hunk 13]
```diff
@@ -0,0 +1,40 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-random.yml:2]: 2021

File: compose-builder/add-device-rest.yml [hunk 14]
```diff
@@ -0,0 +1,41 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-rest.yml:2]: 2021

File: compose-builder/add-device-snmp.yml [hunk 15]
```diff
@@ -0,0 +1,40 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-snmp.yml:2]: 2021

File: compose-builder/add-device-virtual.yml [hunk 16]
```diff
@@ -0,0 +1,40 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/add-device-virtual.yml:2]: 2021

File: compose-builder/add-modbus-simulator.yml [hunk 17]
```diff
@@ -0,0 +1,15 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-modbus-simulator.yml:1]: copyright header

File: compose-builder/add-mqtt-broker.yml [hunk 18]
```diff
@@ -0,0 +1,15 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-mqtt-broker.yml:1]: copyright header

File: compose-builder/add-mqtt-messagebus.yml [hunk 19]
```diff
@@ -0,0 +1,48 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel
```
Comment (jim-wang-intel) [inline compose-builder/add-mqtt-messagebus.yml:2]: 2021

File: compose-builder/add-taf-app-services.yml [hunk 20]
```diff
@@ -0,0 +1,51 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-taf-app-services.yml:1]: copyright header

File: compose-builder/add-taf-device-services-mods.yml [hunk 21]
```diff
@@ -0,0 +1,16 @@
+version: '3.7'
```
Comment (jim-wang-intel) [inline compose-builder/add-taf-device-services-mods.yml:1]: copyright header

File: compose-builder/common-security.env [hunk 22]
```diff
@@ -0,0 +1,24 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel
```
Comment (jim-wang-intel) [inline compose-builder/common-security.env:2]: 2021

File: compose-builder/common.env [hunk 23]
```diff
@@ -0,0 +1,32 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel
```
Comment (jim-wang-intel) [inline compose-builder/common.env:2]: 2021

File: compose-builder/docker-compose-base.yml [hunk 24]
```diff
@@ -0,0 +1,252 @@
...
+#  *
+#  * @author: Jim White, Dell
+#  * @author: Andre Srinivasan, Redis Labs
+#  * @author: Leonard Goodell
+#  * EdgeX Foundry, Hanoi, version "master"
+#  * added: May, 2020
+#  *******************************************************************************/
+
+# NOTE:  this Docker Compose file does not contain the security services - namely the API Gateway
+# and Secret Store
```
Comment (jim-wang-intel) [inline compose-builder/docker-compose-base.yml:23]: the comments about security service is kind of out-of-date; not just api gateway and secret store but also other security services.

File: compose-builder/docker-compose-ui.yml [hunk 25]
```diff
@@ -0,0 +1,38 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline compose-builder/docker-compose-ui.yml:2]: 2021

File: compose-builder/gen-header [hunk 26]
```diff
@@ -0,0 +1,26 @@
...
+#  *******************************************************************************/
+#
+#
+#
+# ************************ This is a generated compose file ****************************
+#
+# DO NOT MAKE CHANGES that are intended to be permanent to EdgeX developer-scripts repo.
+#
+# Permanent changes can be made to the source compose files located in the compose-builder folder
+# at the top level of the developer-scripts repo.
```
Comment (jim-wang-intel) [inline compose-builder/gen-header:23]: this comment out-of-date, no longer developer-scripts

File: releases/pre-release/Makefile [hunk 27]
```diff
@@ -0,0 +1,71 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel
```
Comment (jim-wang-intel) [inline releases/pre-release/Makefile:2]: 2021

File: releases/pre-release/docker-compose-portainer.yml [hunk 28]
```diff
@@ -0,0 +1,34 @@
+# /*******************************************************************************
+#  * Copyright 2020 Intel Corporation.
```
Comment (jim-wang-intel) [inline releases/pre-release/docker-compose-portainer.yml:2]: 2021

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
some minor clean-ups

File: compose-builder/gen-header [hunk 29]
```diff
@@ -0,0 +1,26 @@
...
+#  * the License.
+#  *
+#  * EdgeX Foundry, Hanoi, version "master"
+#  *******************************************************************************/
+#
+#
+#
+# ************************ This is a generated compose file ****************************
+#
+# DO NOT MAKE CHANGES that are intended to be permanent to EdgeX developer-scripts repo.
```
Comment (jim-wang-intel) [inline compose-builder/gen-header:20]: out-of-date

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by jpwhitemn — APPROVED** [review]

**Pull request merged**